lib/modules/apostrophe-doc-type-manager/public/js/chooser.js

```diff
@@ -324,6 +325,50 @@ apos.define('apostrophe-doc-type-manager-chooser', {
         position: { collision: 'flip' }
       });
     };
+
+    self.enableSortable = function () {
+      var sortableOptions = {
+        handle: '[data-apos-sort-item]',
```

Comment:
How is this attribute actually getting into the markup somewhere?

Reply:
https://github.com/apostrophecms/apostrophe/pull/1385/files#diff-adc89237501d9c40a3a33cf2c1c963f2R7 

and the like